docs/QUICK_START.md

```diff
@@ -0,0 +1,202 @@
+
+Direct USB connection is another useful option for both PCs and mobile devices which support OTG USB.
+- Unfortunately this option is not available with the FrSky R9M.
+- This option does not work out of the box with the recommended MatekSys Tx modules for this use case because the USB port on these modules is blocked when installed in a radio JR bay.
+- It is possible to solder a type A USB cable directly to the USB pads on the back of the MatekSys Tx modules.
+- If you do this, you will need to flash the "siktelem" version of the firmware for your MatekSys Tx module.
+- If you solder a cable to these pads, you must then also use the soldered USB cable for flashing and not connect anything to the built-in type C jack.
+
+## 8. Radio calibration
+
+There are many calibration and setup steps required for ArduPilot on a new aircraft.  Here, we mention just the RC calibration step.  Not only because RC calibration needs to be repeated when switching to mLRS from a different RC system, but also because of a quirk you may encounter.
+
+- If you normally use QGroundControl as your ground station, be aware that it does not properly handle Radio calibration when using "rc_override".
+- If you did not connect sbus or CRSF RC output from your receiver to your flight controller, you should use another ground station such as Mission Planner or APM Planner 2.0 to perform radio calibration.
+- You can, of course, connect either of these ground stations via your mLRS Tx module or directly via USB to your flight controller.
+- This is a good time to configure and check the operation of any flight modes and switches you might want.
+
+## 9. Bench and range testing
+
+Before your first flight with mLRS, it is good to do some additional testing to ensure everything is working as expected.
+
+- Remove props and power everything up.
+- Verify the Rx and Tx modules connect by observing the LEDs.
+- Use the mLRS Lua script or CLI to set the Rx and Tx power to the lowest setting.
+- Start your ground station and verify that parameters download in a reasonable time depending on mode (6-60 seconds).
+- Verify that everything you expect such as attitude, GPS position, battery voltage, etc is being displayed correctly by both the ground station and Yaapu telemetry if you use it.  If you don't use Yaapu or don't see what you expect, check that you see the expected telemetry sensors on your radio (MDL -> TELEMETRY -> Discover new sensors)
+- Ensure you can change flight modes and arm/disarm the aircraft from both your RC radio and your ground station.
+- Perform a "walking range test" by leaving the aircraft behind while you walk away with the transmitter and ground station.
+- If mLRS doesn't work as expected, see the [Troubleshooting](TROUBLE.md) section.
+- Don't forget to set the Rx and Tx power back to a reasonable level before your first flight.
```

Comment:
dito as before

note: the MAtekSys cannot be set to a really low power

Reply:
Here, Rx and Tx are the page labels they will see or part of the parameter name they will type when they change it.

Reply:
no one will understand that fine distinction line
call it "... the "Rx Power" and "Tx Power" parameters ..."
or "... the receiver and Tx module power ..."